_data/accessibility-tests/identifier.yml

```diff
@@ -0,0 +1,103 @@
+title: Identifier
+component_status: pass
+test_items:
+# General tests
+  - summary: Icon identifies external links.
+    summary_additional: An external link in the identifier will have an icon next to it that indicates the user will be taken away from the current site. 
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.4
+  - summary: Color alone is not used to convey information.
+    summary_additional: When viewing the identifier, you never have to rely on color alone to understand information.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.1
+  - summary: Identifier icons meets color contrast requirements. 
+    summary_additional: When you use a color contrast checker to view the identifier, contrast between the icon and adjacent colors is at least 3:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.3
+  - summary: Identifier text meets color contrast requirements.  
+    summary_additional: When you use a color contrast checker to view the identifier, contrast between the text color and background color is at least 4.5:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.3
+  - summary: Links provide direct location description.  
+    summary_additional: You can read the text of any link in the identifier to understand where the link will take you.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.4
```

Comment:
**blocking issue:** It looks like this criteria has not yet been included in [wcag-details.html](https://github.com/uswds/uswds-site/blob/main/_includes/accessibility-tests/wcag-details.html). Can you add 2.4.4 to that file?  (There should be info in the process doc related to "wcag_criterion")

Reply:
Looks good! Thanks!